doc/ssl/SSL_CTX_set_tlsext_status_cb.pod

```diff
@@ -16,8 +21,10 @@ Certificate Status Request functions
  long SSL_CTX_set_tlsext_status_arg(SSL_CTX *ctx, void *arg);
 
  long SSL_CTX_set_tlsext_status_type(SSL_CTX *ctx, int type);
+ long SSL_CTX_get_tlsext_status_type(SSL_CTX *ctx);
 
  long SSL_set_tlsext_status_type(SSL *s, int type);
+ long SSL_get_tlsext_status_type(SSL *s);
```

Comment:
I wonder if dropping SSL_ctrl could be included in the 1.1.0 scope. Though I guess that would be a pretty change.